front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -257,25 +284,32 @@ public boolean supportsVersioning() {
   }
 
   @Override
-  public <T extends Timestamped> T
-  loadObject(ObjectType objectType, String objectKey)
-         throws NotFoundException {
+  public <T extends Timestamped> T loadObject(ObjectType objectType, String objectKey) throws NotFoundException {
     String path = keyToPath(objectKey, objectType.group);
     try {
-      StorageObject storageObject = timeExecute(loadTimer, obj_api.get(bucketName, path));
-      T item = deserialize(storageObject, (Class<T>) objectType.clazz, true);
-      item.setLastModified(storageObject.getUpdated().getValue());
+      StorageObject[] storageObjectHolder = new StorageObject[1];
```

Comment:
What does this accomplish over writing `StorageObject storageObject = null`, and then assigning to that? Oh - does this have to be effectively final?